src/tiledimage.js

```diff
@@ -189,7 +194,7 @@ $.TiledImage = function( options ) {
         _arrayCacheMap: [],    // array cache to avoid constant re-creation and GC overload
         _isBlending:    false, // Are any tiles still being blended?
         _wasBlending:   false, // Were any tiles blending before the last draw?
-        _isTainted:     false, // Has a Tile been found with tainted data?
+        _issues:        {},    // Has a the image was marked as problematic by some entity (usually a drawer)?
```

Comment:
Grammar on this comment. Also, it's multiple, right? So it's not just a flag (which is what the current comment implies).